flow/designs/sky130hs/ibex/config.mk

```diff
@@ -44,3 +44,5 @@ export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
 export CORE_UTILIZATION = 45
 export PLACE_DENSITY_LB_ADDON = 0.2
 export TNS_END_PERCENT = 100
+
+export GPL_TARGET_RC = 1.25
```

Comment:
Please assign yourself an issue to remove the need for this if you haven't already.

Reply:
Already did that on #1916